README.md

```diff
@@ -1,22 +1,16 @@
 # 🎉**Welcome to AIMon Rely**
 
-AIMon Rely is a state-of-the-art system consisting of multiple models for detecting LLM quality issues during offline evaluations and continuous production monitoring. We offer
-hallucination metrics that is fast, reliable and cost-effective. We also support additional metrics such as
-completeness, conciseness and toxicity. 
+AIMon Rely helps developers build, ship, and monitor LLM Apps more confidently and reliably with its state-of-the-art, multi-model system for detecting LLM quality issues. It helps seamlessly with both offline evaluations and continuous production monitoring. AIMon Rely offers fast, reliable, and cost-effective hallucination detection and related metrics. It also supports other important quality metrics such as completeness, conciseness, and toxicity. Read our [blog post](https://aimon.ai/blogs/introducing-rely) for more details.
```

Comment:
Lets remove ` and related metrics` since we talk about the quality metrics in the next sentence.

Reply:
done